lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1623,6 +1631,7 @@ def cmd_sessions(*args)
             if session.respond_to?(:response_timeout)
               last_known_timeout = session.response_timeout
               session.response_timeout = response_timeout
+              session.on_error_proc = log_on_timeout_error("Send timed out. Timeout currently #{session.response_timeout} seconds, you can configure this with sessions -C <cmd> --timeout <value>")
```

Comment:
Let's make this pop a bit more so that user's eyes are attracted to the 'solution'

```suggestion
              session.on_error_proc = log_on_timeout_error("Send timed out. Timeout currently #{session.response_timeout} seconds, you can configure this with %grnsessions -C <cmd> --timeout <value>%clr")
```

Likewise with the below message